core/tests/webdriverio_utils/waitFor.js

```diff
@@ -36,6 +36,16 @@ var alertToBePresent = async() => {
     });
 };
 
+var urlToBe = async function(url) {
```

Comment:
Add a comment explaining the use case of this?

Reply:
Done.